eZ/Bundle/EzPublishCoreBundle/Translation/Collector.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * File containing the Collector class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Translation;
+
+/**
+ * Retrieves all installed ezplatform translation files.
+ */
+class Collector
```

Comment:
Rather than technically, if you want to put an interface on it I'll name it VendorCollector cause it's suppose to retrieve translations installed in vendor so this open the door to a DbCollector or other.

What do you think ?